requirements.txt

```diff
@@ -12,3 +12,4 @@ pybossa-client
 fuzzywuzzy
 python-Levenshtein
 iso3166
+tox
```

Comment:
We expect `tox` to be installed globally, as it isn't a project requirement. You could tell this on the `Run tests` part of the docs, like:

> To run the tests, you need `tox` installed. We recommend you to install it globally with `pip install -g tox`.